demos/multi_channel/common/multichannel_params.hpp

```diff
@@ -14,6 +14,9 @@ static const char help_message[] = "Print a usage message";
 /// @brief Message for model argument
 static const char face_detection_model_message[] = "Required. Path to an .xml file with a trained model.";
 
+/// @brief Message for model argument
+static const char yolo_model_message[] = "Required. Path to an .xml file with a trained model.";
```

Comment:
I suggest just renaming `face_detection_model_message`->`model_path_message` and remove `yolo_model_message`. But it would require updating two other demos. Alternative solution can be moving this variable into `multichannel_object_detection_demo_yolov3_params.hpp`.